packages/client/lib/sync/fetcher/blockfetcher.ts

```diff
@@ -1,60 +1,45 @@
+const level = require('level')
 import { Fetcher, FetcherOptions } from './fetcher'
 import { Block, BlockBodyBuffer } from '@ethereumjs/block'
-import { BN } from 'ethereumjs-util'
 import { Peer } from '../../net/peer'
 import { EthProtocolMethods } from '../../net/protocol'
-import { Chain } from '../../blockchain'
-
-export interface BlockFetcherOptions extends FetcherOptions {
-  /* Blockchain */
-  chain: Chain
-
-  /* Block number to start fetching from */
-  first: BN
-
-  /* How many blocks to fetch */
-  count: BN
-}
+import VM from '@ethereumjs/vm'
+import { DefaultStateManager } from '@ethereumjs/vm/dist/state'
```

Comment:
Yeah, I have done this for the first time as well 😄 , think it's good that we are finally getting consumers of the `StateManager`, think this will give us a better feeling here. The import e.g. is not very optimal going through `dist`, on the next major release we should likely directly expose through the main `index.ts` file, have added this as a breaking task within a new release v6 (+ friends 😄 ) planning issue #1024.